### DISCUSSION TITLE
More specific about dates in the roadmap

### DISCUSSION THREAD

**Discussion by ghost** [joomla/joomla-cms#32276] [General] [answered]
I think it would be good for Joomla to be more specific about dates in the roadmap. 

Someone who was looking for Joomla! 4 for a larger project in December will decide against it when it becomes more specific in February. 
You can see from the discussion about Bootstap 5 that things can be changed. This is annoying if you are expected Bootstrap 4. 
But if you have no clue at all, then you decide against it right from the start. 
And you don't even read the planning for 4.1 and 4.2 because it's too far away. You can't believe that this will ever happen. Only on a closer look it is clear that it is being developed in parallel. Not many external people do that.

**Comment by dgrammatiko** [comment, marked as answer]
> We expect this to be the last beta of Joomla 4 and to have a Release Candidate available in March.

source: https://www.joomla.org/announcements/release-news/5833-joomla-4-0-0-beta7-and-joomla-3-10-alpha5.html

**Comment by ghost** [comment]
Thank you.
As most extern people look in the roadmap you find via google, it would be good, if this is shown here too: https://developer.joomla.org/roadmap.html#4.
